communication/mailing-list-guidelines.md

```diff
@@ -1,293 +1,270 @@
-  - [Sharing the calendar with the Google Group](#sharing-the-calendar-with-the-google-group)
-  - [Sharing the meeting notes with the Google Group](#sharing-the-meeting-notes-with-the-google-group)
-- [Archive a mailing list](#archive-a-mailing-list)
-
+The Kubernetes mailing list, using Google Groups, functions as the primary means
+of asynchronous communication for the project's 
+[Special Interest Groups (SIG), Working Groups (WG)][], and large subprojects.
 
 ## Code of conduct
 
-The Kubernetes project adheres to the community [Code of Conduct] throughout all
-platforms and includes all communication mediums.
+The Kubernetes project adheres to the community [Code of Conduct][] throughout
+all platforms and includes all communication mediums.
 
 ## Admins
 
-Check the [centralized list of administrators][admins] for contact information.
+Check the [centralized list of administrators][] for contact information.
 
-To connect: Reach out to one of the listed moderators, [mailing list owners],
-the [SIG Contributor Experience mailing list] or the `#sig-contribex` slack
+To connect: Reach out to one of the listed moderators, [mailing list owners][],
+the [SIG Contributor Experience mailing list][] or the #sig-contribex slack
 channel.
 
 ### Mailing list owners
 
-Mailing list owners should include the Chairs for your [SIG or WG][sig-list], 
-or the leads for your subproject, and the below contacts:
+Mailing list owners should include the Chairs for your [SIG or WG][], or the
+leads for your subproject, and the below contacts:
 
-- contributors@kubernetes.io
-
----
+-   contributors@kubernetes.io
 
 ## Moderation
 
 SIG, Working Group, and subproject mailing lists should have the 
-[mailing list owners] as co-owners to the list so that administrative functions 
-can be managed centrally across the project.
+[mailing list owners][] as co-owners to the list so that administrative
+functions can be managed centrally across the project.
 
-Moderation of the SIG/WG/subproject lists is up to that individual 
-SIG/WG/subproject. The admins are there to help facilitate leadership changes, 
+Moderation of the mailing lists is the responsibility of that individual SIG,
+WG, or subproject. Admins are available to help facilitate leadership changes,
 or various other administrative functions.
 
-Users who are violating the [Code of Conduct] or other negative activities
-(like spamming) should be moderated.
-- [Lock the thread immediately] so that people cannot reply to the thread.
-- [Delete the post].
-- In some cases you might need to ban a user from the group, follow
-  [these instructions] on how stop a member from being able to post to the 
-  group.
-  For more technical help on how to use Google Groups, check the [Groups Help]
-  page.
+When a user violates the [Code of Conduct][] or performs other negative
+activities (like spamming or harassment), moderators should:
```

Comment:
we should also add a bullet about contacting the CoCC here.

Reply:
Hey! @jberkus can I confirm what you mean by the CoCC? I'm not sure what that is referring to. Thanks in advance!

Reply:
Code of Conduct Committee

Reply:
@jberkus I addressed this in the latest push!